.github/ISSUE TEMPLATES/node-related-package-releases.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Node-related package releases
+about: Use this template for tracking package releases that pertain to a specific release of `cardano-node`
+title: "Release: for cardano-node [VERSION]"
+---
+
+# Packages and versions released
+
+<!-- Please fill out this table _only_ after actually making the `CHaP`
+releases. Delete from the table below those packages that _did not need_ a new
+version released to `CHaP` -->
+
+| Package | Version |
+|-|-|
+| `base-deriving-via` | |
+| `cardano-binary` | |
+| `test/cardano-binary-test` | |
+| `cardano-crypto-class` | |
+| `cardano-crypto-praos` | |
+| `cardano-crypto-tests` | |
+| `cardano-git-rev` | |
+| `cardano-slotting` | |
+| `cardano-strict-containers` | |
+| `heapwords` | |
+| `measures` | |
+| `orphans-deriving-via` | |
```

Comment:
This is totally unnecessary. It is better to have a bullet point with a check box that asks to list PR(s) to the CHaP repo, because it will have all of this info
```suggestion
```